src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,88 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { User } from '../../types/User';
+import { actions as actionsCurrentTodo } from '../../features/currentTodo';
+import { getUser } from '../../api';
+
+type Props = {
+  setIsLoadingUser: React.Dispatch<React.SetStateAction<boolean>>;
+  isLoadingUser: boolean;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  setIsLoadingUser,
+  isLoadingUser,
+}) => {
+  const dispatch = useAppDispatch();
+  const [user, setUser] = useState<User | null>(null);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const handleCloseModal = () => {
+    dispatch(actionsCurrentTodo.removeTodo());
+  };
+
+  const getDataUser = async () => {
+    try {
+      setIsLoadingUser(true);
+
+      if (currentTodo) {
+        const userFromServer = await getUser(currentTodo?.userId);
+
+        setUser(userFromServer);
+      }
+    } catch (error) {
+    } finally {
+      setIsLoadingUser(false);
+    }
+  };
+
+  useEffect(() => {
+    getDataUser();
+  }, [currentTodo]);
```

Comment:
fix it, please 
<img width="906" alt="image" src="https://github.com/mate-academy/react_redux-list-of-todos/assets/98764641/a05b2190-7901-4d2a-b30a-cbe892208200">